djstripe/models.py

```diff
@@ -320,6 +328,9 @@ class Customer(StripeObject):
     date_purged = models.DateTimeField(null=True, editable=False)
 
     objects = CustomerManager()
+    
+    allow_multiple_subscriptions = ALLOW_MULTIPLE_SUBSCRIPTIONS
+    retain_canceled_subscriptions = RETAIN_CANCELED_SUBSCRIPTIONS
```

Comment:
It's also worth pointing out that, with ALLOW_MULTIPLE_SUBSCRIPTIONS set to False, the previous CurrentSubscription semantics are fully preserved. Further calls to Customer.subscribe() end up just being treated as plan changes/upgrades, and only one Subscription record is ever utilised. RETAIN_CANCELED_SUBSCRIPTIONS has no effect on this.